### PR TITLE
Add 4th chunk of Cobra Examples

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Adds the 4th chunk of Cobra Examples to the CLI help.
One more chunk of 10 to complete.

### CODE REVIEW

File: cmd/podman/restore.go [hunk 1]
```diff
@@ -29,7 +29,9 @@ var (
 			restoreCommand.GlobalFlags = MainGlobalOpts
 			return restoreCmd(&restoreCommand)
 		},
-		Example: "CONTAINER-NAME [CONTAINER-NAME ...]",
+		Example: `podman container restore ctrID
+  podman container restore --latest
+  podman container restore -all`,
```
Comment (mheon) [inline cmd/podman/restore.go]: Missing a `-`?

File: cmd/podman/stats.go [hunk 2]
```diff
@@ -41,7 +41,9 @@ var (
 			statsCommand.GlobalFlags = MainGlobalOpts
 			return statsCmd(&statsCommand)
 		},
-		Example: "",
+		Example: `podman stats -a --no-stream
+  podman stats --no-stream imageID
```
Comment (mheon) [inline cmd/podman/stats.go]: One without --no-stream?

File: cmd/podman/restore.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/restore.go]: yes, fixed.  I was so worried about spaces at the end that I completely missed that.

File: cmd/podman/stats.go (hunk 2 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/stats.go]: done and changed to ctrID from imageID

**Comment by mheon** [comment]
LGTM

**Pull request merged**